tasks/54_roman-numbers-convert/roman-numbers-convert.js

```diff
@@ -0,0 +1,48 @@
+export function convertToRoman(num) {
+  let romanNum = [];
+
+  const thousands = Math.floor(num / 1000);
+  let remainder = num % 1000;
+
+  for (let i=0; i<thousands; i++) {
+    romanNum.push('M');
+  }
+
+  const hundreds = Math.floor( remainder/ 100);
```

Comment:
This logic also repeats 4 times, with just different parameters. Extract to a method.